server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -369,6 +397,16 @@ public void listSourcesShouldReturnWhenHasForwardMapping() throws Exception {
         assertThat(virtualUserTable.listSources(mapping)).contains(source);
     }
 
+    @Test
+    public void listSourcesShouldReturnWhenHasAliasMapping() throws Exception {
```

Comment:
I propose to modify this naming pattern to `listSourcesShouldReturnAliasMappings` (& also for related tests, maybe in a separate commit if need be)